### PR TITLE
WIP: Set Copyright property to match License.txt

### PR DESCRIPTION
**Pull request by tmat**
The `Copyright` property is used by .NET SDK to emit `AssemblyCopyrightAttribute`.

The default copyright set by Arcade SDK is `© Microsoft Corporation.  All rights reserved.`

### CODE REVIEW

**Comment by tmat** [comment]
@ericstj @danmosemsft @richlander I assume assemblies should have the same copyright as specified in the license.txt file, correct?

**Comment by richlander** [comment]
This is the copyright file that we should be using (source and binary) in nearly all cases: https://github.com/dotnet/coreclr/blob/master/LICENSE.TXT

The only case I'm aware of where we'd want a different license is for a couple closed source components, (all relating to diagnostic/debugging scenarios, AFAIK; that's the vs debugger and diasymreader) that we distribute.  I don't know if those relate to Arcade. We can talk about which licenses those should be using if that ends up being important.

**Comment by richlander** [comment]
The default copyright in Arcade is wrong. We should not take this PR and instead fix the default in Arcade. Like I said, there are only a couple components that should use a different license and I don't even know if those use Arcade.

**Comment by mmitche** [comment]
This make it into the nuget package metadata.  I think you're probably right that the default should be changed in Arcade, but we need to make sure this doesn't fail nuget's package validation.

**Comment by richlander** [comment]
What nuget validation are you referring to?

**Comment by mmitche** [comment]
Package validation checks on nuget push.  Signing, author checks, etc.

**Comment by tmat** [comment]
@richlander The default copyright is not wrong. Arcade SDK is not used only for .NET Foundation projects. It's used for many other projects that are not in .NET Foundation. We have two options: 1) report an error when Copyright is not set explicitly, 2) use a safe default and override it in repos that are under .NET Foundation.

**Comment by tmat** [comment]
@richlander BTW Roslyn uses `Copyright (c) Microsoft.` in [sources](https://github.com/dotnet/roslyn/blob/master/src/EditorFeatures/CSharp/AutomaticCompletion/AutomaticLineEnderCommandHandler.cs#L1), assemblies and packages, although it's a .NET Foundation project. It's license is Apache 2.0. So perhaps that needs to be updated?

**Comment by tmat** [comment]
@mmitche There is no validation that checks copyright, other than in SignTool that understands both copyrights.

**Comment by richlander** [comment]
I think the default for Arcade should be open source (.NET Foundation and MIT). That's really the base of this discussion. The Roslyn one is wrong, too.

**Comment by tmat** [comment]
We decided in an offline conversation to require repositories to explicitly choose copyright from a given predefined copyright strings. I'll update this PR accordingly.

File: Directory.Build.props [hunk 1]
```diff
@@ -251,6 +251,10 @@
   </PropertyGroup>
 
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+  
+  <PropertyGroup>
+    <Copyright>© .NET Foundation and Contributors</Copyright>
+  </PropertyGroup>
```
Comment (danmosemsft) [inline Directory.Build.props]: Do we need PackageLicenseExpression as well then?

Comment (tmat) [reply Directory.Build.props]: I have a new PR: https://github.com/dotnet/corefx/pull/35524 will close this one

**Comment by tmat** [comment]
Superseded by https://github.com/dotnet/corefx/pull/35524

**Pull request closed without merging**